tools/harness-thci/OpenThread_BR.py

```diff
@@ -465,14 +481,18 @@ def __getEthGUA(self, filterByPrefix=None):
                 continue
 
             addr = line[1].split('/')[0]
-            addr = ModuleHelper.GetFullIpv6Address(addr).lower()
+            addr = str(ipaddress.IPv6Address(addr.decode()).exploded)
             globalAddrs.append(addr)
 
         if not filterByPrefix:
             return globalAddrs[0]
         else:
+            if filterByPrefix[-2:] != '::':
+                filterByPrefix = '%s::' % filterByPrefix
+            prefix = ipaddress.IPv6Address(filterByPrefix.decode())
             for fullIp in globalAddrs:
-                if fullIp.startswith(filterByPrefix):
+                address = ipaddress.IPv6Address(fullIp.decode())
+                if address.packed[:8] == prefix.packed[:8]:
```

Comment:
```suggestion
                if address in prefix:
```
if prefix is `ipaddress.IPv6Network`, we can simply use `in`.

Reply:
Done.